salt/observer/init.sls

```diff
@@ -127,6 +127,24 @@ configure-app:
             - log-file
             - psql-app-db
 
+#
+#
+#
+
+daily-metrics-import:
+    # don't scrape ejp data outside of prod/adhoc instances
+    {% if pillar.elife.env not in ['prod'] %}
```

Comment:
I would prefer a separate `observer/cron.sls` file to be configured only for `prod` in `top.sls`, to avoid hardcoding environment names.